src/core/payment/SubstituteAndExcludePayToAddressesForSuperCharge.ts

```diff
@@ -34,9 +34,33 @@ export class SubstituteAndExcludePayToAddressesForSuperCharge implements ISubsti
                                     !(transaction.amount <= payoutThreshold),
                             )
                             .map((t) => {
-                                if (t.publicKey == record[0]) t.publicKey = record[1];
+				if (t.publicKey == record[0] && record[1] != 'SPLIT') t.publicKey = record[1];
```

Comment:
I think you have to relax parse - I get an error due to variable column count.

/Users/jrwashburn/code/mina/mina-pool-payout/node_modules/csv-parse/dist/cjs/index.cjs:940
new CsvError('CSV_RECORD_INCONSISTENT_FIELDS_LENGTH', [
^
CsvError: Invalid Record Length: expect 2, got 4 on line 8
at Object.__onRecord (/Users/jrwashburn/code/mina/mina-pool-payout/node_modules/csv-parse/dist/cjs/index.cjs:940:11)
at Object.parse (/Users/jrwashburn/code/mina/mina-pool-payout/node_modules/csv-parse/dist/cjs/index.cjs:825:40)
at Parser._flush (/Users/jrwashburn/code/mina/mina-pool-payout/node_modules/csv-parse/dist/cjs/index.cjs:1336:26)
at Parser.final [as _final] (node:internal/streams/transform:132:10)
at callFinal (node:internal/streams/writable:698:12)
at prefinish (node:internal/streams/writable:710:7)
at finishMaybe (node:internal/streams/writable:720:5)
at Parser.Writable.end (node:internal/streams/writable:634:5)
at ReadStream.onend (node:internal/streams/readable:705:10)
at Object.onceWrapper (node:events:628:28) {
code: 'CSV_RECORD_INCONSISTENT_FIELDS_LENGTH',
bytes: 570,
comment_lines: 0,
empty_lines: 0,
invalid_field_length: 0,
lines: 8,
records: 7,
columns: false,
error: undefined,
header: false,
index: 4,
raw: undefined,
column: 4,
quoting: false,
record: [
'B62qmNbNedwwDB3bLUzgeN6gAYXmnY4U9WBS6Ze92HghtU7x3v7a2sS',
'SPLIT',
'B62qoigHEtJCoZ5ekbGHWyr9hYfc6fkZ2A41h9vvVZuvty9amzEz3yB',
'0.5'
]
}

Reply:
This fixes it on line 28
.pipe(parse({ relax_column_count_more: true, delimiter: '|' }))

Reply:
Fixed by adding an option. Locally I added extra empty delimeters, my bad.